client/state/active-promotions/test/reducer.js

```diff
@@ -15,13 +14,7 @@ import activePromotionsReducer, {
 import { WPCOM_RESPONSE } from './fixture';
 
 describe( 'reducer', () => {
-	let sandbox;
-
-	useSandbox( ( newSandbox ) => {
-		sandbox = newSandbox;
-		// mute off console warn
-		sandbox.stub( console, 'warn' );
-	} );
+	console.warn = jest.fn();
```

Comment:
This will assign a mock to `console.warn` and it will stay assigned there forever? The following should be more reliable:
```js
jest.spyOn(console, 'warn').mockImplementation()
```
in a sense that Jest will automatically restore the original `console.warn` when resetting mocks.

Reply:
I actually gave it a try in 2 separate files, and ran them in a separate it seems like the mocks are being reset when using `jest.fn()` as well. I set `jest.fn()` in the first file, and removed it from the second one, and `console.warn()` was warning correctly in the console for the second one.

Reply:
That's because every test runs in its own VM, parallelized across several worker processes. If you try `jest -i` (alias for `--runInBand`), commonly used in debugging, all tests will run in one VM and I think you'll start to see `console.warn` being permanently overwritten.